### PR TITLE
Reformat the codebase

### PR DESCRIPTION
**Pull request by DellaBitta**
Ran `scripts/format_code.py` on the entire sourcebase.

### CODE REVIEW

File: analytics/ios_headers/FIRUserPropertyNames.h [hunk 1]
```diff
@@ -19,28 +19,31 @@
 ///
```
Comment (alexames) [inline analytics/ios_headers/FIRUserPropertyNames.h:19]: I don't think we should be formatting the obj c headers, should we?

File: app/src/fake/FIRApp.h [hunk 2]
```diff
@@ -21,7 +21,6 @@ extern "C" {
 void FIRAppCreateUsingDefaultOptions(const char* name);
```
Comment (alexames) [inline app/src/fake/FIRApp.h:21]: Another ObjC file I think that we should be leaving alone.

File: app/rest/transport_curl.cc [hunk 3]
```diff
@@ -426,9 +445,7 @@ void BackgroundTransportCurl::CompleteOperation() {
 }
 
 void BackgroundTransportCurl::CheckOk(CURLcode code, const char* msg) {
-  if (code == CURLE_OK) {
-    return;
-  }
+  if (code == CURLE_OK) { return; }
```
Comment (alexames) [inline app/rest/transport_curl.cc:448]: nit: There's a bunch of one-liners like this that scooped up the {}'s. Can we set it up to not one-line {} blocks? If we must one-line them, can we remove the {}'s?

I ran the formatter within google3 on a few test files and it *wasn't* turning the bodies of if statements into one liners if they were in curly brackets. Is this just an inconsistency in formatter rules? Which one is more correct?

File: .clang-format [hunk 4]
```diff
@@ -3,8 +3,12 @@ Standard: Cpp11
 ColumnLimit: 80
```
Comment (alexames) [inline .clang-format:3]: What's the rationale for the changes to this file? It seems to introduce more changes than it preserves. Shouldn't we stick with whatever the defaults of the Google style are?

File: performance/src_ios/fake/FIRHTTPMetric.h [hunk 5]
```diff
@@ -5,18 +5,19 @@
...
-  FIRHTTPMethodPUT NS_SWIFT_NAME(put),
-  FIRHTTPMethodPOST NS_SWIFT_NAME(post),
-  FIRHTTPMethodDELETE NS_SWIFT_NAME(delete),
-  FIRHTTPMethodHEAD NS_SWIFT_NAME(head),
-  FIRHTTPMethodPATCH NS_SWIFT_NAME(patch),
-  FIRHTTPMethodOPTIONS NS_SWIFT_NAME(options),
-  FIRHTTPMethodTRACE NS_SWIFT_NAME(trace),
-  FIRHTTPMethodCONNECT NS_SWIFT_NAME(connect)
-} NS_SWIFT_NAME(HTTPMethod);
+      FIRHTTPMethodPUT NS_SWIFT_NAME(put),
```
Comment (alexames) [inline performance/src_ios/fake/FIRHTTPMetric.h:8]: hahaha what is going on here.

(probably a side effect of this being an ObjC file)

**Comment by DellaBitta** [comment]
Closing this PR for a new formatting run.

File: app/rest/transport_curl.cc (hunk 3 above)
Comment (jonsimantov) [reply app/rest/transport_curl.cc:448]: +1

File: .clang-format (hunk 4 above)
Comment (jonsimantov) [reply .clang-format:3]: +1
